message-index/messages/GHC-20125/index.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Missing Fields
+summary: Initialization of record with missing fields
+severity: warning
+introduced: 9.6.1
+---
+The initialization of a record is missing one or more (lazy) fields.
+When constructing a record using labeled fields, all fields which are not explicitly initialized are implicitly initialized with a term bottom which will panic at runtime if the field is accessed.
```

Comment:
The wording seems to use `initialized` too much. What about?

```
When constructing a record using labeled fields, all fields which are not explicitly stated are implicitly initialized with a bottom term. This means that accessing the field results in a panic.
```

Reply:
Done in 16d3e90